### PR TITLE
Completed html-tables exercise - Andrew Almeida

### PR DESCRIPTION
**Pull request by A-MAlmeida**
38FF1ECD-C7ED-4802-B15C-5BDAD746F19A_1_105_c

### CODE REVIEW

File: html-tables/index.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title> HTML TABLES </title>
+</head>
+
+<body>
+
+  <h1> Student Grade Table </h1>
+
+  <table>
+
```
Comment (Zhiwutian) [inline html-tables/index.html:15]: Almost there! The only thing to add is the `thead` and `tbody` elements for the headings and data

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update!

File: html-tables/index.html (hunk 1 above)
Comment (A-MAlmeida) [reply html-tables/index.html:17]: done!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**